### DISCUSSION TITLE
How to set listener for pull request merged?

### DISCUSSION THREAD

**Discussion by nazreen** [octokit/octokit.js#2220] [Q&A] [answered]
so we can listen to `pull_request.closed` but that can either be a pull request that was rejected or merged, right? how do I check if the pull request was merged?

**Comment by wolfy1339** [comment, marked as answer]
Simply check that the payload's `pull_request.merged` property is `true`
https://github.com/octokit/webhooks/blob/1c4b64c7d23cadd2406a12a5cf7b8afdf1054246/payload-types/schema.d.ts#L4429-L4447
